genRSS.py

```diff
@@ -117,48 +110,41 @@ def build_item(link, title, guid = None, description="", pub_date=None, indent =
     '''
     Generate an RSS 2.0 item and return it as a string.
 
-    Parameters
-    ----------
-    link : string
-           URL of the item.
+    Args:
+        link (string): URL of the item.
+
+        title (string): Title of the item.
+
+        guid (string): Unique identifier of the item. If no guid is given, link is used as the identifier.
+            Default = None.
 
-    title : string
-            Title of the item.
+       description (string): Description of the item.
+            Default = ""
 
-    guid : string
-           Unique identifier of the item. If no guid is given, link is used as the identifier.
-           Default = None.
 
-   description : string
-                 Description of the item.
-                 Default = ""
+    pubDate (string): Date of publication of the item. Should follow the RFC 822 format,
+            otherwise the feed will not pass a validator.
+            This method does not (yet) check the compatibility of pubDate.
+            Here are a few examples of correct RFC 822 dates:
 
-    pub_date : string
-              Date of publication of the item. Should follow the RFC 822 format,
-              otherwise the feed will not pass a validator.
-              This method does not (yet) check the compatibility of pub_date.
-              Here are a few examples of correct RFC 822 dates:
 
-              - "Wed, 02 Oct 2002 08:00:00 EST"
-              - "Mon, 22 Dec 2014 18:30:00 +0000"
+            - "Wed, 02 Oct 2002 08:00:00 EST"
+            - "Mon, 22 Dec 2014 18:30:00 +0000"
 
-              You can use the following code to gererate an RFC 822 valid time:
-              time.strftime("%a, %d %b %Y %H:%M:%S +0000", time.localtime(time.time()))
-              Default = None (no pubDate tag will be added to the generated item)
+            You can use the following code to gererate an RFC 822 valid time:
+            time.strftime("%a, %d %b %Y %H:%M:%S +0000", time.localtime(time.time()))
+            Default = None (no pubDate tag will be added to the generated item)
 
-    indent : string
-             A string of whitespaces used to indent the elements of the item.
-             3 * len(indent) whitespaces will be left before <guid>, <link>, <title> and <description>
-             and 2 * len(indent) before item.
+        indent (string): A string of whitespaces used to indent the elements of the item.
```

Comment:
You should use `str` instead of `string` here (and for argument of the same type) as `str` is a class and is the expected type.